v2/client_test.go

```diff
@@ -84,6 +96,16 @@ func doHTTP(t *testing.T, name string, checks httpChecks, reaction httpReaction)
 			return nil, walkingGhostErr
 		}
 
+		if len(checks.headers) > 0 {
+			for k, v := range checks.headers {
+				actualValue := request.Header.Get(k)
+				if e, a := v, actualValue; e != a {
+					t.Errorf("%v: unexpected header value for key %v; expected %v, got %v", name, k, e, a)
```

Comment:
only if you're in here anyway... it might be nice to change the second+ %v's to %q' so they're quoted. It makes it easier to read and reduces the changes of those values looking like normal English words and people not parsing the sentence correctly.